app/mutator.py

```diff
@@ -139,7 +139,6 @@ def isDirty(self):
   def isSafeToWrite(self):
     if not os.path.exists(self.fullPath):
       return True
-    self.isReadOnly = not os.access(self.fullPath, os.W_OK)
```

Comment:
Try making a file read only; open and edit it; try to save and get error; then change it to writable (in another terminal window); then come back to ci_edit; I'd like it to no longer show the file as read only.

(and I think this PR will still say [RO] because we only set it in fileLoad)

Reply:
I think in this case we might need a background thread that just constantly polls the file for its permissions, and if the file doesnt exist, we would have to poll the directory that it is in. I think currently the [RO] will only change when you try to save, but since I don't think we can detect whether the user switched focus from some other window to ci_edit without a keylogger, I think we'd just have to check periodically.